### PR TITLE
Update wam.md

### PR DESCRIPTION
**Pull request by bgavrilMS**
1. Drop obscure option "ListOperatingSystemAccounts"
2. Showcase SSO with Windows

### CODE REVIEW

File: msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md [hunk 1]
```diff
@@ -59,7 +59,29 @@ IPublicClientApplication app =
     .WithBroker(options)
     .Build();
 
-var authResult = await app.AcquireTokenInteractive(new List<string>() { "User.Read" }).ExecuteAsync();
+IAccount existingAccount = await FetchExistingAccountFromCache().ConfigureAwait(false);
```
Comment (localden) [inline msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md:62]: @bgavrilMS we likely need to document what `FetchExistingAccountFromCache` - it's referenced in the code snippet but not clear what the customer needs to do here or where the function implementation lives.

**Review by localden — CHANGES_REQUESTED** [review]

File: msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md [hunk 2]
```diff
@@ -59,7 +59,29 @@ IPublicClientApplication app =
...
+AuthenticationResult result = null;
+
+try
+{
+    // Try to sing-in the previously signed-in account
```
Comment (gladjohn) [inline msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md:67]:
```suggestion
    // Try to sign-in the previously signed-in account
```

File: msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md [hunk 3]
```diff
@@ -59,7 +59,29 @@ IPublicClientApplication app =
...
+    else
+    {    
+        result = await _pca.AcquireTokenSilent(scopes, PublicClientApplication.OperatingSystemAccount)
+                            .ExecuteAsync();
+    }
+}
+// Can't get a token silently, go interactive
+catch (MsalUiRequiredException ex)
+{
+    result = await app.AcquireTokenInteractive(new List<string>() { "User.Read" }).ExecuteAsync();
```
Comment (gladjohn) [inline msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md:82]:
```suggestion
    result = await app.AcquireTokenInteractive(new List<string>() { scopes }).ExecuteAsync();
```

File: msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md (hunk 1 above)
Comment (gladjohn) [reply msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md:62]: maybe we can add the getAccounts API and also how to get the OS Accounts, thought we already had it

Comment (gladjohn) [reply msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md:62]: nm, I see we are dropping it

File: msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md [hunk 4]
```diff
@@ -59,7 +59,29 @@ IPublicClientApplication app =
...
+AuthenticationResult result = null;
+
+try
+{
+    // Try to sing-in the previously signed-in account
+    if (existingAccount != null)
+    {
+        result = await _pca.AcquireTokenSilent(scopes, existingAccount).ExecuteAsync();
+    }
+    // If it does not exist, try to sign in with the Windows account
```
Comment (gladjohn) [inline msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md:72]:
```suggestion
    // If it does not exist, try to sign in with the Windows account
```

maybe we can say the account that is already signed in to Windows (currently logged in user or something like that)

**Review by gladjohn — APPROVED** [review]

**Pull request merged**